src/sqlfluff/dialects/dialect_sparksql.py

```diff
@@ -2685,3 +2686,24 @@ class DescribeDetailStatementSegment(BaseSegment):
             Ref("TableReferenceSegment"),
         ),
     )
+
+
+class GenerateManifestFileStatementSegment(BaseSegment):
+    """A statement to `GENERATE` manifest files for a Delta Table.
+
+    https://docs.delta.io/latest/delta-utility.html#generate-a-manifest-file
+    """
+
+    type = "generate_manifest_file_statement"
+
+    match_grammar: Matchable = Sequence(
+        "GENERATE",
+        "symlink_format_manifest",
```

Comment:
Note that, as this is a keyword, it will be subject to keyword capitalisation rules (e.g. L010). I presume this should always be lowercase?

you could replace this with this (and remove from Keyword file) to avoid this:

```suggestion
        StringParser("symlink_format_manifest", CodeSegment, name="symlink_format_manifest"),
```